### DISCUSSION TITLE
Is it possible to create a template for describing documentation in git folders?

### DISCUSSION THREAD

**Discussion by EvgeniyaTsib** [community/community#67069] [New to GitHub]
Tell me, is it possible to create a template or take an existing template in Git in order to place project documentation in it?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @EvgeniyaTsib , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by frg** [comment]
Yes of course! Github templates can be utilized in any way you see fit.
